src/main/java/frc/robot/commands/tank/PlebAutonSequence.java

```diff
@@ -50,67 +57,73 @@ public void initialize() {
+        private final GRTNetworkTableEntry shotsCompletedEntry = shuffleboardTab.addEntry("Shots completed", shotsCompleted);
+        private final GRTNetworkTableEntry completedEntry = shuffleboardTab.addEntry("Completed", complete);
+
+        @Override
+        public void initialize() {
+            turretSubsystem.setDriverOverrideFlywheel(true);
+            autonTimer.start();
+            shotsCompleted = 0;
+        }
 
-        turretSubsystem.setDriverOverrideFlywheel(true);
-    }
+        @Override
+        public void execute() {
+            forceTimerEntry.setValue(forceTimer.get());
+            shotsCompletedEntry.setValue(shotsCompleted);
+
+            // We are done driving if: we are beyond 55 inches of our starting position,
+            // or 8 seconds have passed.
+            boolean doneDriving = tankSubsystem.distance(start) > Units.inchesToMeters(55) 
+                || autonTimer.hasElapsed(8);
+
+            // If we're not done driving, drive forwards at 0.4 power
+            // If we are done driving, start the timer to force a shot.
+            tankSubsystem.setCarDrivePowers(!doneDriving ? 0.4 : 0, 0);
+            if (doneDriving) forceTimer.start();
+
+            // This runs once immediately and after every successful shot.
+            // If we're not requesting a shot, request one and reset the force timer.
+            // End the command after shooting twice.
+            if (!internalSubsystem.getShotRequested()) {
+                if (doneDriving && shotsCompleted == 2) {
+                    intakeSubsystem.setPosition(IntakePosition.RAISED);
+                    completedEntry.setValue(true);
+                    complete = true;
+                }
+                forceTimer.reset();
+                internalSubsystem.requestShot();
+                shotsCompleted++;
+            }
 
-    @Override
-    public void execute() {
-        if (complete) return;
-
-        // We are done driving if: we are beyond 55 inches of our starting position,
-        // or 8 seconds have passed.
-        boolean doneDriving = tankSubsystem.distance(start) > Units.inchesToMeters(55) 
-            || autonTimer.hasElapsed(8);
-
-        // If we're not done driving the set distance or time, drive forwards at 0.4 power
-        tankSubsystem.setCarDrivePowers(!doneDriving ? 0.4 : 0, 0);
-
-        // This runs once immediately and after every successful shot.
-        // If we're not requesting a shot, request one and reset the force timer.
-        // End the command after shooting twice.
-        if (!internalSubsystem.getShotRequested()) {
-            if (doneDriving && shotsRequested == 2) {
-                intakeSubsystem.setPosition(IntakePosition.RAISED);
-                tankSubsystem.setCarDrivePowers(0.4, 0);
-                finalTimer.start();
-                complete = true;
+            // Force a shot if we haven't shot in 4 seconds
+            if (forceTimer.hasElapsed(4)) {
```

Comment:
8 seconds driving + 8 seconds of forcing for two balls is *just* too much time in a worst case scenario

Reply:
Perhaps the 8 seconds of driving time is the issue? I kinda don't want to decrease the force timer out of fear that the first shot may be forced out too early.

Reply:
I think shooting two balls at once fixes this